apps/studio/components/interfaces/Settings/Addons/IPv4SidePanel.tsx

```diff
@@ -0,0 +1,281 @@
+      })
+      onClose()
+    },
+    onError: (error) => {
+      ui.setNotification({
+        error,
+        category: 'error',
+        message: `Unable to enable IPv4: ${error.message}`,
+      })
+    },
+  })
+  const { mutate: removeAddon, isLoading: isRemoving } = useProjectAddonRemoveMutation({
+    onSuccess: () => {
+      ui.setNotification({
+        category: 'success',
+        message: `Successfully disabled IPv4.`,
+      })
+      onClose()
+    },
+    onError: (error) => {
+      ui.setNotification({
+        error,
+        category: 'error',
+        message: `Unable to disable IPv4: ${error.message}`,
+      })
+    },
+  })
+  const isSubmitting = isUpdating || isRemoving
+
+  const subscriptionIpV4Option = (addons?.selected_addons ?? []).find(
+    (addon) => addon.type === 'ipv4'
+  )
+  const availableOptions =
+    (addons?.available_addons ?? []).find((addon) => addon.type === 'ipv4')?.variants ?? []
+
+  const isFreePlan = subscription?.plan?.id === 'free'
+  const hasChanges = selectedOption !== (subscriptionIpV4Option?.variant.identifier ?? 'ipv4_none')
+  const selectedIPv4 = availableOptions.find((option) => option.identifier === selectedOption)
+
+  useEffect(() => {
+    if (visible) {
+      if (subscriptionIpV4Option !== undefined) {
+        setSelectedOption(subscriptionIpV4Option.variant.identifier)
+      } else {
+        setSelectedOption('ipv4_none')
+      }
+      Telemetry.sendActivity(
+        {
+          activity: 'Side Panel Viewed',
+          source: 'Dashboard',
+          data: {
+            title: 'IPv4',
+            section: 'Add ons',
+          },
+          projectRef,
+        },
+        router
+      )
+    }
+  }, [visible, isLoading])
+
+  const onConfirm = async () => {
+    if (!projectRef) return console.error('Project ref is required')
+    if (selectedOption === 'ipv4_none' && subscriptionIpV4Option !== undefined) {
+      removeAddon({ projectRef, variant: subscriptionIpV4Option.variant.identifier })
+    } else {
+      updateAddon({ projectRef, type: 'ipv4', variant: selectedOption as AddonVariantId })
+    }
+  }
+
+  return (
+    <SidePanel
+      size="large"
+      visible={visible}
+      onCancel={onClose}
+      onConfirm={onConfirm}
+      loading={isLoading || isSubmitting}
+      disabled={isFreePlan || isLoading || !hasChanges || isSubmitting || !canUpdateIPv4}
+      tooltip={
+        isFreePlan
+          ? 'Unable to enable IPv4 on a free plan'
+          : !canUpdateIPv4
+          ? 'You do not have permission to update IPv4'
+          : undefined
+      }
+      header={
+        <div className="flex items-center justify-between">
+          <h4>IPv4</h4>
+          <Button asChild type="default" icon={<IconExternalLink strokeWidth={1.5} />}>
+            <Link
+              href="https://github.com/orgs/supabase/discussions/17817"
+              target="_blank"
+              rel="noreferrer"
+            >
+              About IPv4
```

Comment:
the CTA sounds weird here

the link is to the deprecation notice but this "About IPv4" sounds like a doc about IPv4 address?
```suggestion
              About IPv4 deprecation
```

Reply:
This is just a temporary link until we moved it to docs, adjusted the copy for now